### ISSUE TITLE
HelpOptionAttribute is not implemented

### ISSUE THREAD

**Issue by bangonkali** [cosmo0/commandline#1]
The HelpOption is not implemented for Version 2.0.

**Comment by galmok** [comment]
Is any part of the help system working?
Using AutoBuild, it was easy to have basic help, but with the HelpOption gone, I don't know how to get any help. 
I feel the help system is a major part of this library and I would be sad to see it go. Fortunately, 1.9 is still around and working well.

**Comment by cosmo0** [comment]
For 2.0, it seems the help system is completely broken.

As I said, I will not work on it until I have a real need for it. However, I will accept pull requests into my fork.

**Comment by galmok** [comment]
Fair enough. I just reverted to the 1.9.74 release instead. It is working fine.
